### PR TITLE
Move acceptance test feature files to different folder structure

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
1) Put acceptance tests suite folders in the ``tests/acceptance/features`` folder.
2) Remove ``_feature`` from the end of the folder name (it is already in a ``features`` folder in step 1)
3) Prefix the name with ``api`` so we know (for now) it is a suite of ``api`` tests. 

## Related Issue
https://github.com/owncloud/QA/issues/517

## Motivation and Context
The Behat suites for the API tests are stored directly under ``tests/acceptance`` with folder names like ``federation_features``. The "other" features are all just in the ``features`` folder, which is one of the suites.

The Behat suite for the UI tests are stored inside ``tests/ui/features`` with names like ``manageUsersGroups`` ``renameFiles`` etc.

This needs to be standardized as we move towards merging the Behat folder structures.

## How Has This Been Tested?
CI knows

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] test refactoring

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by phil-davis** [comment]
For discussion abut which way we should go.

**Review by individual-it — APPROVED** [review]

**Comment by phil-davis** [comment]
@SergioBertolinSG your opinion about this will be good to hear. There are a few ways we could structure this, we just need to choose one and then be consistent.

File: tests/acceptance/config/behat.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ default:
   autoload:
     '': %paths.base%/../features/bootstrap
   suites:
-    default:
+    apiOther:
```
Comment (SergioBertolinSG) [inline tests/acceptance/config/behat.yml]: Could we rename this? apiOther sounds like forgotten and not liked tests, but they are actually the most important.

apiMain or apiGeneral for example?

Comment (phil-davis) [reply tests/acceptance/config/behat.yml]: Good idea, I will do.

**Comment by phil-davis** [comment]
Backport stable10 #30680

**Pull request merged**